locations/spiders/tmobile_pl.py

```diff
@@ -0,0 +1,21 @@
+from scrapy import Spider
+
+from locations.dict_parser import DictParser
+from locations.hours import DAYS_PL, OpeningHours
+
+
+class TMobilePLSpider(Spider):
+    name = "tmobile_pl"
+    item_attributes = {"brand": "T-Mobile International", "brand_wikidata": "Q327634"}
+    start_urls = ["https://www.t-mobile.pl/c/_bffapi/sdr-shops/v1/shops"]
+
+    def parse(self, response, **kwargs):
+        for feature in response.json():
```

Comment:
The source data has a field `feature["address"]["street_prefix"]` that is often populated, but which we don't appear to be extracting. Should we be extracting `item["street"] = " ".join(filter(None, [feature["address"].get("street_prefix"), feature["address"].get("street")]))`

Reply:
All stores have `street_prefix` set.
Output of `http GET 'https://www.t-mobile.pl/c/_bffapi/sdr-shops/v1/shops' | jq '.[].address.street_prefix' | sort | uniq -c | sort -h`
```
      1 "Os."
     45 "Al."
     50 "Pl."
    713 "Ul."
```
Os. is osiedle -> neighbourhood
Al. is aleja -> alley
Pl. is plac -> square
Ul. is ulica -> street

In OpenStreetMap in Poland we omit ulica from name, rest is kept.

I added `street_prefix` based on above in 28c70df44b8cee5e86ce34a558e5b937b1d15d7d